lib/src/revset.rs

```diff
@@ -339,10 +339,7 @@ impl RevsetExpression {
 
     /// Descendants of `self`, including `self`.
     pub fn descendants(self: &Rc<RevsetExpression>) -> Rc<RevsetExpression> {
-        Rc::new(RevsetExpression::Descendants {
-            roots: self.clone(),
-            generation: GENERATION_RANGE_FULL,
-        })
+        self.descendants_range(GENERATION_RANGE_FULL)
```

Comment:
nit: can you also update `descendants_at()`?

Reply:
You mean make it call `descendants_range` the way `ancestors_at` calls `ancestors_range`? Sure.